src/bp/ui-studio/src/web/components/Layout/StatusBar/LangSwitcher.tsx

```diff
@@ -0,0 +1,127 @@
+import { lang } from 'botpress/shared'
+import _ from 'lodash'
+import React, { Fragment } from 'react'
+import { Dropdown } from 'react-bootstrap'
```

Comment:
bootstrap 😭  !! you can leave it there though as we will take this away and implement it in second level toolbar